store-conf/conf/log4j.properties.production

```diff
@@ -207,8 +207,7 @@ logger.memcached.name = net.spy.memcached
 logger.memcached.level = WARN
 
 %%comment VAR:!zimbraLogHostname%%logger.slogger.name = zimbra.slogger
-%%comment VAR:!zimbraLogHostname%%logger.slogger.level=INFO
-%%uncomment VAR:!zimbraLogHostname%%logger.slogger.level=ERROR
+%%comment VAR:!zimbraLogHostname%%logger.slogger.level = info
```

Comment:
I would still suggest the following config to restore previous (log4j v1) behaviour:
```
logger.slogger.name = zimbra.slogger
%%comment VAR:!zimbraLogHostname%%logger.slogger.level = info
%%uncomment VAR:!zimbraLogHostname%%logger.slogger.level = ERROR
logger.slogger.additivity = false
%%comment VAR:!zimbraLogHostname%%logger.slogger.appenderRef.SLOGGER.ref = sloggerAppender
```
Otherwise it keeps spamming mailbox.log with statistics logging when zimbraLogHostname is not set (the default).

Reply:
zimbraLogHostname is setup in most system as it copies hostname by default.

Reply:
Looking at zmsetup.pl, that's only the case when zimbra-logger is installed.

Reply:
I guess both are same

Reply:
This was the old config:
```
log4j.additivity.zimbra.slogger=FALSE
%%comment VAR:!zimbraLogHostname%%log4j.logger.zimbra.slogger=INFO,SLOGGER
%%uncomment VAR:!zimbraLogHostname%%log4j.logger.zimbra.slogger=ERROR
```

Hence slogger level was set to ERROR when zimbraLogHostname wasn't set (and these logs go to mailbox.log).

In the current v2 config, the slogger block gets all commented when zimbraLogHostname isn't set, so everything goes to mailbox.log at default loglevel, spamming the log with unwanted statistics.

(In the old v2 config, only the level=ERROR line got uncommented, but this broke the config since "slogger" definition was still commented.)

Reply:
@ghen2  check now